__test__/App.spec.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { createStore, combineReducers } from 'redux';
+import { BrowserRouter, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import LoginPage from '@pages/loginPage';
+import HomePage from '@pages/homePage';
+import NotFoundPage from '@pages/notFoundPage';
+import Menu from '../src/components/userMenu';
+import Footer from '../src/components/Footer/index';
+import Auth from '../src/components/auth';
+
+import App from '../src/routes/AppRouter';
+
+const auth = (state = {}, action) => {
+  switch (action.type) {
+    case 'LOGIN_SUCCESS':
+      return {
+        isLoginPending: false,
+        isAuthenticated: !action.user,
+        user: action.user
+      };
+    default:
+      return state;
+  }
+};
+
+describe('Application test', () => {
+  let store;
+  beforeEach(() => {
+    store = createStore(
+      combineReducers({
+        auth
+      })
+    );
+  });
+  it('should render index page', () => {
+    const comp = (
+      <Provider store={store}>
+        <BrowserRouter>
+          <Switch>
+            <App dispatch={jest.fn()} />
+          </Switch>
+        </BrowserRouter>
+      </Provider>
+    );
+    const wrapper = mount(comp);
+
+    expect(wrapper.find('Home')).toBeTruthy();
+  });
+  it('should not crash app', () => {
+    const wrapper = mount(<App />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+  it('should work fine on Login Page', () => {
+    const wrapper = mount(
+      <BrowserRouter>
+        <LoginPage />
+      </BrowserRouter>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+  it('should work fine Home Page', () => {
+    const wrapper = mount(
+      <BrowserRouter>
+        <HomePage />
+      </BrowserRouter>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('should work fine Home Page', () => {
```

Comment:
please kindly change the test of this it blocks to describe what the test does.
the snapshot shows NotFoundPage. kindly look into that.

Reply:
Thanks for the feedback. This has been fixed